stytch/b2b/models/sessions.py

```diff
@@ -114,3 +114,25 @@ class RevokeResponse(ResponseBase):
     """Response type for `Sessions.revoke`.
     Fields:
     """  # noqa
+
+
+# MANUAL(AuthenticateM2MJWTLocalResponse)(TYPES)
+class AuthenticateM2MJWTLocalResponse(ResponseBase):
+    """Response type for `Sessions.authenticate_m2m_jwt_local`.
+    Fields:
+      - sub: The subject that the JWT is intended for.
+      - scope: A list of scopes granted, separated by spaces.
+      - custom_claims: A dict of custom claims of the JWT.
+    """  # noqa
+
+    sub: str
+    scope: str
+    custom_claims: Optional[Dict[str, Any]] = None
+
+    def __init__(self, sub, scope, custom_claims):
+        self.sub = sub
+        self.scope = scope
+        self.custom_claims = custom_claims
```

Comment:
1. Add typing for **everything**
2. But this method is unnecessary -- anything inheriting from `pydantic.BaseModel` will automatically be parseable and get an `__init__` method for free
3. But also this isn't a `ResponseBase` -- you're not going to have a `status_code` or `request_id`. Make this inherit from `pydantic.BaseModel` instead